sqlalchemy_kusto/dbapi.py

```diff
@@ -281,13 +282,16 @@ def _escape(value) -> str:
 
         if value == "*":
             return value
-        if isinstance(value, str):
+        elif isinstance(value, str):
             return "'{}'".format(value.replace("'", "''"))
-        if isinstance(value, bool):
+        elif isinstance(value, bool):
             return "TRUE" if value else "FALSE"
-        if isinstance(value, (int, float)):
+        elif isinstance(value, (int, float)):
             return str(value)
-        if isinstance(value, (list, tuple)):
+        elif isinstance(value, (list, tuple)):
             return ", ".join(Cursor._escape(element) for element in value)
+        elif isinstance(value, datetime):
+            value = value.isoformat()
+            return "'{}'".format(value)
```

Comment:
It looks like sqlalchemy by default replaces the T but I tested this with raw queries in the following formats and they all work with the server im connecting to right now.
```
    "2020-01-01 01:01:01",
    "2020-01-01 01:01",
    "2020-01-01 01:01:01.000001",
    "2020-01-01T01:01:01.000001",
    "2020-01-01T01:01:01.000001",
    "2020-01-01T01:01:01.000001+00:00",
    "2020-01-01 01:01:01.000001+00:00",
    "2020-01-01T01:01:01.000001+00:00Z",
    "2020-01-01 01:01:01.000001+00:00Z"
```